### PR TITLE
dropdown done

### PR DESCRIPTION
**Pull request by theVivek-kumar**
drop-down done

### CODE REVIEW

File: drop-menu/drop-menu.css [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    position: absolute;
+    background-color: #f9f9f9;
+    min-width: 160px;
+    box-shadow: 0px 8px 16px 0px rgba(46, 46, 46, 0.2);
+    z-index: 1;
+}
+
+.dropdown-content a {
+    color:var(--BACKGROUND_COLOR);
+    padding: 12px 16px;
```
Comment (Adarsh-Patel-Dev) [inline drop-menu/drop-menu.css:35]: use rem here

File: drop-menu/drop-menu.css [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    text-decoration: none;
+    display: block;
+    font-size: 1.3rem;
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+}
+
+.dropdown-content a:hover {
+    background-color: #3e9ea1bb
```
Comment (Adarsh-Patel-Dev) [inline drop-menu/drop-menu.css:43]: use color var here

File: drop-menu/drop-menu.html [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+                                <a href="#">GIT</a>
+                                <a href="#">LINKDIN</a>
+                            </div>
+                        </div></div>
+
+                    </div>
+                 
+                  
+                    
+
```
Comment (Adarsh-Patel-Dev) [inline drop-menu/drop-menu.html:123]: remove extra spaces here

**Review by Adarsh-Patel-Dev — APPROVED** [review]

**Review by ritikaawasthy — APPROVED** [review]
looks good just be consistent with the units

**Pull request merged**